kafka-samples/kstreams-join-user-clicks-and-region/pom.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-starter-parent</artifactId>
+		<version>2.1.3.RELEASE</version>
+	</parent>
+
+	<groupId>org.springframework.cloud</groupId>
+	<artifactId>kstreams-join-user-clicks-and-region</artifactId>
+	<version>1.0.0.BUILD-SNAPSHOT</version>
+	<packaging>jar</packaging>
+
+	<name>kstreams-join-user-clicks-and-region</name>
+	<description>Demo project for Kafka Streams Table Join</description>
+
+
+	<dependencies>
+
+		<!-- This dependency adds Spring Cloud Stream Binder for Kafka Streams -->
+		<dependency>
+			<groupId>org.springframework.cloud</groupId>
+			<artifactId>spring-cloud-stream-binder-kafka-streams</artifactId>
+			<version>2.1.1.RELEASE</version>
```

Comment:
Why can't we use the spring-cloud BOM to manage the dependency instead of hardcoding here?

Reply:
I deliberately use the hard-coded dependency instead of a common BOM from the parent as these samples should be re-usable as is. Most of the cases, users would just copy the directory and try to make some changes which result in incomplete dependency management due to the parent POM/BOM missing. What do you think?

Reply:
That is fine in that case. What are we going to do next time we have to update the sample with a new boot parent? Are we going to change all those individual dependencies as well? Most often, those dependencies will also have to change with the parent version change, especially if we upgrade the parent to a new major version.

Reply:
I see your point here as this sample might become stale after few releases. I will apply the Spring Cloud BOM in each project (which makes them still re-usable)